src/components/Link/__tests__/Link.spec.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { render, RenderResult, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import { Link } from '../Link';
+
+describe('<Link> that renders a link', () => {
+    const href = 'https://textkernel.com';
+    let view: RenderResult;
+
+    beforeEach(() => {
+        view = render(<Link href={href} />);
+    });
+
+    it('should render nothing if no children are provided', () => {
+        expect(view.container).toMatchSnapshot();
+        expect(view.container).toBeEmptyDOMElement();
+    });
+
+    it('should render children when it is 0', () => {
+        view.rerender(<Link href={href}>{0}</Link>);
+
+        const { children } = view.container;
```

Comment:
what is the advantage of this new pattern for checks instead of the ones we used until now? e.g. `getByText` or `toHaveTextContent` (https://github.com/testing-library/jest-dom#tohavetextcontent)

Reply:
:+1: